bitnami/apache/Chart.yaml

```
@@ -38,4 +38,4 @@ maintainers:
 name: apache
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/apache
-version: 11.3.22
+version: 11.3.23
```

As you are adding a new parameter, we need to bump the chart to a new minor version.

```suggestion
version: 11.4.0
```